### ISSUE TITLE
Sweaty feet odor

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10551]
**Preferred term label:**

**Synonyms**
sweaty foot-like odor
smell of “dirty socks”

**Definition (free text, please give PubMed ID)**
PMID: 16602101 A “smell specialist” suggested that the unusual odor associated with their acute episodes was likely a short chain fatty acid, and isovaleric acid was identified in patient plasma through the then novel approach of gas chromatography followed by mass spectrometry.
(...)Neonatal symptoms are non-specific and include poor feeding, vomiting, decreased level of consciousness, and seizures. Infants may develop hypothermia and appear to be dehydrated. A characteristic **smell of “dirty socks”** may be present when the patient is acutely sick though, unlike other organic acidemias, the urine has no odor since the unconjugated isovaleric acid responsible for the odor is not excreted in urine in appreciable quantity. The odor may be best appreciated in body sweat or cerumen from the ear.

PMID: 38484105 Individuals with clinical manifestations of isovaleric acidemia (IVA) have either classic IVA identified on newborn screening or classic IVA with a later diagnosis due to a missed diagnosis or later onset of clinical manifestations. Classic IVA is characterized by acute metabolic decompensations (vomiting, poor feeding, lethargy, hypotonia, seizures, and a distinct **odor of sweaty feet**).

**Parent term (use hpo.jax.org/app)**
Body odor HP:0500001

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Isovaleric acidemia ORPHA:33

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001063
- label: Sweaty foot-like odor
- parents: [Body odor (HP:0500001)](https://hpo.jax.org/app/browse/term/HP:0500001)
- pmid: PMID:33521113|PMID:38484105|PMID:16602101
- def: A characteristic smell said to resemble the odor of dirty socks.
- comment: This feature is characteristic of isoveleric acidemia.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10551
- synonyms:
